netflix.py

```diff
@@ -0,0 +1,45 @@
+import dab.applications
+import dab.input
+import config
+
+# The user must be logged into Netflix to perform the Play Movie test
+NETFLIX_TEST_CASES = [
+# This is your RDK UI shell, replace callsign if different
+#    ("applications/exit", f'{{"appId": "GravityApp", "background": true}}', dab.applications.exit, 10000, "GravityApp Suspend"),
```

Comment:
What's this line doing here?

Reply:
Hi @bgy36ww, It's a quick fix to avoid mess on TV screen when launching Netflix. When Netflix the only app then we don't need to suspend the UI (or other apps) but it will be unusual case. I'll provide more info in comments below.

Reply:
Oh, I see what you mean, the formatting is a bit hard to read.